Microsoft.DotNet.Cli.sln

```diff
@@ -1473,24 +1474,24 @@ Global
 		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Debug|x64.Build.0 = Debug|Any CPU
 		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Debug|x86.ActiveCfg = Debug|Any CPU
 		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Debug|x86.Build.0 = Debug|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|Any CPU.ActiveCfg = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|Any CPU.Build.0 = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|x64.ActiveCfg = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|x64.Build.0 = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|x86.ActiveCfg = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|x86.Build.0 = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|Any CPU.Build.0 = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|x64.ActiveCfg = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|x64.Build.0 = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|x86.ActiveCfg = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|x86.Build.0 = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.RelWithDebInfo|Any CPU.ActiveCfg = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.RelWithDebInfo|Any CPU.Build.0 = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.RelWithDebInfo|x64.ActiveCfg = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.RelWithDebInfo|x64.Build.0 = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.RelWithDebInfo|x86.ActiveCfg = Release|Any CPU
-		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.RelWithDebInfo|x86.Build.0 = Release|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|Any CPU.ActiveCfg = Debug|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|Any CPU.Build.0 = Debug|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|x64.ActiveCfg = Debug|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|x64.Build.0 = Debug|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|x86.ActiveCfg = Debug|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.MinSizeRel|x86.Build.0 = Debug|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|Any CPU.ActiveCfg = Debug|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|Any CPU.Build.0 = Debug|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|x64.ActiveCfg = Debug|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|x64.Build.0 = Debug|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|x86.ActiveCfg = Debug|Any CPU
+		{CACA427D-5A71-45E6-88DC-3E2DB6C4D52D}.Release|x86.Build.0 = Debug|Any CPU
```

Comment:
I've seen this happen before. Either a VS bug or a misconfiguration on our part somewhere. This is say to build some projects in Debug for release. Please revert.